template.tpl

```diff
@@ -767,104 +759,217 @@ ___TEMPLATE_PARAMETERS___
+                "simpleValueType": true,
+                "defaultValue": true
+              },
               {
-                "displayValue": "Page Loads (default)",
-                "value": "pageLoads"
+                "type": "CHECKBOX",
+                "name": "deviceManufacturer",
+                "checkboxText": "Enrich device_manufacturer using client user agent enrichment",
+                "simpleValueType": true,
+                "defaultValue": true
               },
               {
-                "displayValue": "Only with attribution changes",
-                "value": "attribution"
+                "type": "CHECKBOX",
+                "name": "deviceModel",
+                "checkboxText": "Enrich device_model using client user agent enrichment",
+                "simpleValueType": true,
+                "defaultValue": true
               }
             ],
             "enablingConditions": [
               {
-                "paramName": "attributionPageViewTracking",
-                "type": "EQUALS",
-                "paramValue": true
+                "paramName": "uaParser",
+                "paramValue": true,
+                "type": "EQUALS"
               }
-            ],
-            "displayName": "Page View trigger",
-            "defaultValue": "pageLoads",
-            "simpleValueType": true,
-            "name": "attributionPageViewTrackOn",
-            "type": "SELECT"
-          },
+            ]
+          }
+        ]
+      },
+      {
+        "help": "Check this box to enable default event tracking. \u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sdks/browser-2/#tracking-default-events\"\u003eRead more\u003c/a\u003e.",
+        "defaultValue": true,
+        "simpleValueType": true,
+        "name": "defaultEventTracking",
+        "checkboxText": "Enable default event tracking",
+        "type": "CHECKBOX",
+        "subParams": [
           {
-            "help": "Choose whether to disable automatic Page View tracking upon history changes, to track \u003cstrong\u003eall\u003c/strong\u003e history events (pushState, popstate...), or to collect Page Views only when the page path changes. \u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sdks/marketing-analytics-browser/#single-page-app-page-view-tracking\"\u003eRead more\u003c/a\u003e.",
-            "selectItems": [
+            "type": "GROUP",
+            "name": "defaultEventTrackingOptions",
+            "displayName": "Default Event Tracking Options",
+            "groupStyle": "NO_ZIPPY",
+            "subParams": [
+              {
+                "type": "CHECKBOX",
+                "name": "detAttribution",
+                "checkboxText": "Enable attribution tracking",
+                "simpleValueType": true,
+                "subParams": [
+                  {
+                    "type": "GROUP",
+                    "name": "attributionOptions",
+                    "displayName": "",
```

Comment:
is the displayName here empty intentionally?

Reply:
<img width="1136" alt="Screen Shot 2023-07-27 at 1 24 29 PM" src="https://github.com/amplitude/amplitude-browser-sdk-gtm-template/assets/13028329/8154f78f-2e75-4bd8-9f1d-ebed1883ac99">
It's a little bit redundant if we add a display name here. I will leave this as empty.